include/nbl/asset/IShader.h

```diff
@@ -75,6 +75,11 @@ class NBL_API IShader : public virtual core::IReferenceCounted
 	std::string m_filepathHint;
 };
 
+inline IShader::E_SHADER_STAGE operator|(IShader::E_SHADER_STAGE lhs,
+                                         IShader::E_SHADER_STAGE rhs) {
+  return static_cast<IShader::E_SHADER_STAGE>(static_cast<uint32_t>(lhs) |
+                                     static_cast<uint32_t>(rhs));
+}
```

Comment:
@Erfan-Ahmadi @achalpandeyy don't we have a macro for that?

Reply:
> Not yet

should I merge then?

Reply:
NO.
Oh this PR is only this?
do this manually for now. (Portability Use bitflags like previously)
I'll add it soon.

Reply:
> (Portability Use bitflags like previously)

I second this.

Am I missing some context? The intended usage should be `core::bitflag<IShader::E_SHADER_STAGE>`
In #345 we do the same.

Reply:
No it's just about being able to do bitflag(A|B|C) instead of bitflag(A)|B|C
Also for implicit conversions you can just do A|B|C which is much nicer.

But I'm gonna have this for our bitflag enums via a define instead of manual implementation for each of them.

Reply:
It's just a convenience thing.